source/_docs/guides/drupal-8-composer-no-ci.md

```diff
@@ -0,0 +1,194 @@
+
+**Note** this doc uses the `andrew-drops-8-composer` site slug. This will need to be replaced with your unique site slug.
+
+# Creating the Pantheon Site
+
+To begin, we’ll want to start a brand new Drupal 8 site on Pantheon from our empty upstream. This upstream is different from the Drupal 8 upstream in that it does not come with any Drupal files. As such, you must use Composer to download Drupal.
+
+Create a new Pantheon site with an empty upstream
+
+```
+terminus site:create andrew-drops-8-composer 'Andrew Drops 8 Composer' empty
+```
+
+**Note** you can also add the `--org` argument to `terminus site:create` if you would like the site to be part of an organization. See `terminus site:create -h` for details and help.
+
+# Cloning example-drops-8-composer locally
+
+Instead of setting up `composer.json` manually it is easier to start with the [`example-drops-8-composer`](https://github.com/pantheon-systems/example-drops-8-composer) repository.
+
+First, clone the `example-drops-8-composer` repository locally.
+
+```
+git clone git@github.com:pantheon-systems/example-drops-8-composer.git andrew-drops-8-composer
+```
+
+Change into the cloned directory.
+
+```
+cd andrew-drops-8-composer
+
+ls
+```
+
+## Updating the git remote URL
+
+Find the git URL for the Pantheon site created earlier.
+
+```
+terminus connection:info andrew-drops-8-composer.dev --field=git_url
+```
+
+Update the git remote to use the Pantheon site git URL returned rather than the `example-drops-8-composer` GitHub URL.
+
+```
+git remote set-url origin ssh://codeserver.dev.SITE_UUD@codeserver.dev.SITE_UUD.drush.in:2222/~/repository.git
+```
+
+# Removing Automation Pieces
+`example-drops-8-composer` was built for automated testing and continuous integration. Since that functionality won't be removed you can safely remove the directories and files below.
+
+* `script/github`
+* `script/gitlab`
+* `.circleci`
+* `tests`
+
+# Updating `composer.json`
+There are also dependencies defined in `composer.json`, such as testing suites, that will not be used. To avoid downloading those additional, unused dependencies make the following modifications to `composer.json`.
+
+* Remove all dependencies in the `require-dev` section
+* Remove the `lint` sub-section of the `scripts` section
+* Remove the `code-sniff` sub-section of the `scripts` section
+* Remove the `unit-test` sub-section of the `scripts` section
+* Remove the `"find .circleci/scripts/pantheon/ -type f | xargs chmod 755",` line of the `post-update-cmd` sub-section of the `scripts` section
+* Remove the `"find tests/scripts/ -type f | xargs chmod 755"` line of the `post-update-cmd` sub-section of the `scripts` section
+* Remove the comma at the end of the `"DrupalProject\\composer\\ScriptHandler::createRequiredFiles"` line of the `post-update-cmd` sub-section of the `scripts` section to make the JSON valid
+
+The `scripts` section should now be:
```

Comment:
@mcdwayne added a snippet of what the `scripts` section should look like after the modifications. Does this help?